static/app/components/events/eventTags/index.tsx

```diff
@@ -76,6 +77,14 @@ export function EventTags({event, organization, projectSlug, location}: Props) {
     }
   }, [event, organization]);
 
+  useEffect(() => {
+    const mechanism = event.tags.find(tag => tag.key === 'mechanism')?.value;
+    const transaction = Sentry.getCurrentHub().getScope()?.getTransaction();
+    if (mechanism && transaction) {
+      transaction.tags.hasMechanism = mechanism;
```

Comment:
Is there any official guideline on whether tags are supposed to be camel cased, like I have right now, or whether they should be changed to underscore separated (`has_mechanism`)?

Reply:
it's not consistent anywhere, something we should work toward improving though.

I think `hasMechanism` is good enough for now.